robot/impl/resource_manager.go

```diff
@@ -36,6 +37,7 @@ func init() {
 }
 
 var (
+	resourceCloseTimeout    = 30 * time.Second
```

Comment:
minor

```suggestion
const resourceCloseTimeout    = 30 * time.Second

var (
```

Reply:
Ah I do modify that variable in the test so that's why it was `var`. If we want it as `const` I think the test would have to take 30s.

Reply:
sorry! yeah just saw that when I scrolled further :sweat: